files/zh-cn/web/javascript/reference/statements/index.md

```diff
@@ -44,48 +44,113 @@ JavaScript 应用程序是由许多语法正确的语句组成的。单个语句
 - {{jsxref("Statements/function", "function")}}
   - : 声明一个指定参数的函数。
 - {{jsxref("Statements/function*", "function*")}}
-  - : 生成器函数使[迭代器](/zh-CN/docs/Web/JavaScript/Guide/The_Iterator_protocol)更容易使用。
+  - : 生成器函数使编写[迭代器](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols)更容易。
 - {{jsxref("Statements/async_function", "async function")}}
   - : 使用指定的参数声明一个异步函数。
-- {{jsxref("Statements/return", "return")}}
-  - : 指定函数的返回值。
+- {{jsxref("Statements/async_function*", "async function*")}}
+  - : 异步生成器函数使编写异步[迭代器](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols)更容易。
 - {{jsxref("Statements/class", "class")}}
   - : 声明一个类。
 
-### 迭代器
+### 迭代
 
 - {{jsxref("Statements/do...while", "do...while")}}
-  - : 创建一个循环来执行语句，直到该语句条件表达式的值为 false。先执行语句，再执行条件表达式，该语句至少会执行一次。
+  - : 创建一个循环来执行指定语句，直到测试条件的求值为假。先执行语句，再对条件表达式求值，指定语句至少会执行一次。
 - {{jsxref("Statements/for", "for")}}
-  - : 创建一个由 3 个可选的表达式组成的循环，该循环用括号包裹，分号分割，并在循环体中执行语句。
+  - : 创建一个由三个可选的表达式（用括号包裹，分号分割）组成的循环，并在循环体中执行语句。
 - {{jsxref("Statements/for...in", "for...in")}}
   - : 无序遍历对象的可枚举属性。语句针对每个唯一的属性。
 - {{jsxref("Statements/for...of", "for...of")}}
-  - : 遍历可迭代的对象（包括 {{jsxref("Global_Objects/Array","数组")}}、类数组对象、[迭代器和生成器](/zh-CN/docs/JavaScript/Guide/Iterators_and_generators))，对每个不同属性的属性，调用一个自定义的有执行语句的迭代钩子。
+  - : 遍历可迭代对象（包括 {{jsxref("Global_Objects/Array","数组")}}、类数组对象、[迭代器和生成器](/zh-CN/docs/Web/JavaScript/Guide/Iterators_and_generators)），对每个不同的属性的值，调用一个自定义的带有要执行的语句的迭代钩子。
 - {{jsxref("Statements/for-await...of", "for await...of")}}
-  - : 在异步可迭代对象、类数组对象、[迭代器和生成器](/zh-CN/docs/JavaScript/Guide/Iterators_and_generators)上迭代，调用自定义迭代钩子，其中包含要为每个不同属性的值执行的语句。
+  - : 遍历异步可迭代对象、类数组对象、[迭代器和生成器](/zh-CN/docs/Web/JavaScript/Guide/Iterators_and_generators)，对每个不同的属性的值，调用一个自定义的带有要执行的语句的迭代钩子。
 - {{jsxref("Statements/while", "while")}}
-  - : 创建一个循环语句，循环会一直持续到该语句条件表达式的值为 false。先执行条件表达式，然后执行语句。
+  - : 创建一个执行指定语句的循环，直到测试条件的求值为假。先对条件进行求值，再执行语句。
 
 ### 其他
 
+- {{jsxref("Statements/Empty", "空语句", "", 1)}}
+  - : 空语句，用于不在 JavaScript 期望语句语法的地方提供语句。
```

Comment:
The English content does not use the term "statement syntax", it only emphasizes "syntax"